### PR TITLE
Cookstyle fixes for the private-chef cookbook

### PR DESCRIPTION
**Pull request by tas50**
Some automated fixes from Cookstyle. I haven't tested this at all so be sure to do that before shipping this.

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: omnibus/files/private-chef-cookbooks/private-chef/resources/pg_sqitch.rb [hunk 1]
```diff
@@ -15,7 +15,6 @@
 #
 default_action :deploy
 
-attribute :name,           kind_of: String, required: true, name_attribute: true
```
Comment (tas50) [inline omnibus/files/private-chef-cookbooks/private-chef/resources/pg_sqitch.rb:18]: it's possible we were abusing the name property here, but if that's the case we need to stop calling this resource with a name property specified.

**Comment by PrajaktaPurohit** [comment]
https://buildkite.com/chef/chef-chef-server-master-omnibus-adhoc/builds/1351

**Comment by PrajaktaPurohit** [comment]
https://buildkite.com/chef/chef-chef-server-master-integration-test/builds/194

File: omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb [hunk 2]
```diff
@@ -68,8 +68,11 @@
 default['private_chef']['addons']['ubuntu_supported_codenames'] =
```
Comment (tas50) [inline omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb:68]: @PrajaktaPurohit I'm real confused by this part of the cookbook code:

Lucid: 10.04
Precise: 12.04
Trusty: 14.04

All of those are EOL and will never be encountered so this logic means we're always going to fill in Ubuntu 10.04, which seems very wrong. I'm also not seeing where this attribute is actually used in the cookbook anymore.

Comment (PrajaktaPurohit) [reply omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb:68]: Good catch! That should point to the latest.

**Pull request closed without merging**